packages/devtools_app/test/support/utils.dart

```diff
@@ -42,6 +44,46 @@ Future<Timeline> loadNetworkProfileTimeline() async {
   return Timeline.parse(httpTestData);
 }
 
+Future<TreemapNode> loadInstructionSizesJson() async {
+  const testDataPath = '../devtools_testing/lib/support/treemap_test_data.json';
+  final treemapTestData = jsonDecode(
+    await File(testDataPath).readAsString(),
+  );
+  final processedTestData = treemapFromJson(treemapTestData);
+  processedTestData['n'] = 'Root';
+
+  return generateTree(processedTestData);
+}
+
+// TODO(peterdjlee): Remove this method once code size PR (#2107) lands.
+/// Builds a tree with [TreemapNode] from [treeJson] which represents
+/// the hierarchical structure of the tree.
+TreemapNode generateTree(Map<String, dynamic> treeJson) {
```

Comment:
This code also lives in `CodeSizeController` which I was planning on using after deleting this method once the code size PR (#2107) lands. Instead, should I be taking this method out to something like `treemap_utils.dart`?

Reply:
yeah. let's add here and figure out how to clean up in your code size cl